articles/active-directory/develop/msal-android-b2c.md

```diff
@@ -3,15 +3,15 @@ title: Azure AD B2C (MSAL Android) | Azure
 titleSuffix: Microsoft identity platform
 description: Learn about specific considerations when using Azure AD B2C with the Microsoft Authentication Library for Android (MSAL.Android)
 services: active-directory
-author: brianmel
+author: iambmelt
 manager: CelesteDG
 
 ms.service: active-directory
 ms.subservice: develop
 ms.topic: conceptual
 ms.workload: identity
 ms.date: 9/18/2019
-ms.author: brianmel 
+ms.author: iambmelt 
```

Comment:
Also here

Reply:
This should be the author's Microsoft alias.

Reply:
Great - fixed in a998d9e9028715cf5520bfb445fedb52e6bf4112